### PR TITLE
Fix a memset that doesn't zero enough bytes.

### PR DESCRIPTION
**Pull request by jamesjer**

### CODE REVIEW

File: src/base/abci/abc.c [hunk 1]
```diff
@@ -8025,7 +8025,7 @@ int Abc_CommandExact( Abc_Frame_t * pAbc, int argc, char ** argv )
     if ( argc == globalUtilOptind )
         goto usage;
 
-    memset( pTruth, 0, 64 );
+    memset( pTruth, 0, 64 * sizeof(word) );
```
Comment (kristofferkoch) [inline src/base/abci/abc.c]: I also got a warning for this while trying out gcc-9. However, is it more idiomatic with `64 * sizeof(pThruth[0])`?
Of course, the `64` is not nice in itself, but one thing at the time :)

**Comment by kristofferkoch** [comment]
This seems to be addressed by #31 as well.

**Comment by jamesjer** [comment]
Ah, you're right.  I missed #31.  You should probably merge that one and close this one.  As for `sizeof(pTruth[0])`, I've seen it done both ways.  That way does have the advantage of automatically being correct if the base type of the array is changed.

**Comment by kristofferkoch** [comment]
I'm just a random guy on the internet :) This project seems to not have any changes for a long time. Maybe it should be forked and maintained, e.g. by the yosys team?

**Comment by alanminko** [comment]
This project is still active.   Thank you for the pull request.

**Pull request merged**